### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by Christopher-Ho**
JIRA Ticket: https://jira.oicr.on.ca/browse/GLT-3648

- [ X ] Updates release notes
- [ X ] Updates developer documentation

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -50,17 +50,30 @@ list of instruments:
...
+            "/run2"
+        ]
       }
     ]
 
+The ignoreSubdirectories field is used to exclude subdirectories from processing. For example, a 
+known bad run or a test folder holding temporary files like /test/mytestrun. This is optional 
+and in cases where ignoreSubdirectories is not included in the configuration file, all 
+subdirectories under that instrument path will be processed by Run Scanner. For some examples of 
+how to use this feature, check out the tests inside `SchedulerTest`.
```
Comment (avarsava) [inline docs/installation.md:75]: The people installing and running Run Scanner are often not Java developers. Nowhere else in the docs do we recommend looking at source code for explanations of features. (And also, unit tests are in general not a good way to understand features because of the outsized focus on edge cases)

File: docs/installation.md [hunk 2]
```diff
@@ -50,17 +50,30 @@ list of instruments:
...
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/test/run1",
+            "/run2"
+        ]
       }
     ]
 
+The ignoreSubdirectories field is used to exclude subdirectories from processing. For example, a 
+known bad run or a test folder holding temporary files like /test/mytestrun. This is optional
```
Comment (avarsava) [inline docs/installation.md:72]: This `/test/mytestrun` isn't connected to anything and so isn't a meaningful example. Either reference the `/test/run1` in your example config, or just end at "or a test folder holding temporary files."

File: docs/installation.md [hunk 3]
```diff
@@ -50,17 +50,30 @@ list of instruments:
         "platformType": "ILLUMINA",
         "name": "default",
         "timeZone": "America/Toronto",
-        "parameters": {}
+        "parameters": {},
+        "ignoreSubdirectories": [
+            "/run1"
```
Comment (avarsava) [inline docs/installation.md:55]: is the preceding `/` a requirement or will `run1` sans `/` work?

Comment (avarsava) [reply docs/installation.md:55]: Then probably write the examples without `/`. To a new user, these might look like absolute paths despite the 'subdirectories' name.

**Review by avarsava — APPROVED** [review]

**Review by sinry-d — APPROVED** [review]
